src/main.h

```diff
@@ -114,6 +114,14 @@ static const unsigned int WITNESS_WRITE_INTERVAL = 10 * 60;
 static const unsigned int WITNESS_WRITE_UPDATES = 10000;
 /** Maximum length of reject messages. */
 static const unsigned int MAX_REJECT_MESSAGE_LENGTH = 111;
+/** Average delay between local address broadcasts in seconds. */
+static const unsigned int AVG_LOCAL_ADDRESS_BROADCAST_INTERVAL = 24 * 24 * 60;
+/** Average delay between peer address broadcasts in seconds. */
+static const unsigned int AVG_ADDRESS_BROADCAST_INTERVAL = 30;
+/** Average delay between trickled inventory broadcasts in seconds.
+ *  Blocks, whitelisted receivers, and a random 25% of transactions bypass this. */
+static const unsigned int AVG_INVENTORY_BROADCAST_INTERVAL = 5;
```

Comment:
This ties into the inherited-from-upstream "trickle bypass", whereby `inv` messages are broadcast immediately for a random 25% of transactions. I presume this was to prevent a "thundering herd" problem with either low-fee transaction flooding, or where a transaction being broadcast to several well-connected peers simultaneously suddenly multiplies the traffic on the network with duplicative data. Having a random subset of transactions be slightly delayed would make it more likely that the transaction was relayed by someone else before you needed to.

However, the next PR I'll be backporting is https://github.com/bitcoin/bitcoin/pull/7840, which eliminates the tx trickle bypass entirely for both overhead (removing many size-1 `inv` messages) and (what upstream calls) privacy (not announcing txs the moment you receive them). The bypass is replaced by halving this time for outbound peers (i.e. peers our node connected to), so in practice tx bundles will get announced on average 2.5s apart.